src/leafletwidget/widget.ts

```diff
@@ -131,7 +150,10 @@ class MapWidget extends Widget {
     this.update();
   }
 
-  private _geojsonString = '';
+  private _fitBounds = true;
+  private _width = 0;
```

Comment:
I want `_width` to initially be falsey so _fitLayerBounds only happens after a resize event (and I consider something that explicitly has width 0 as needing to still be fitted anyway...)